src/components/Badge.js

```diff
@@ -226,6 +256,13 @@ export const Badge = React.forwardRef(
 )
 
 Badge.propTypes = {
+  /**
+   * Specify variant
+   */
+  variant: PropTypes.oneOf(['primary', 'secondary']),
```

Comment:
nitpick
Perhaps this should rather be "primary" (primary indicating the primary color is used) and "light" (indicating that white colors are getting used). Primary and secondary might be ok for now, but it gets less and less useful as we start adding more variations and it doesnt really hint what make the variant different

Reply:
Hmm, I was trying to match the button props, since the badge is really a glorified button.